lib/metanorma/cli.rb

```diff
@@ -32,7 +32,8 @@ def self.start(arguments)
       end
 
       Metanorma::Cli::Command.start(arguments)
-
+    rescue Interrupt, SignalException => _e
+      UI.say("Good Bye!")
```

Comment:
Let's say "Process cancelled, exiting."

Reply:
Sure thing, since the PR is already merged so I will change that one in a next one 👍🏼